core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -406,6 +380,40 @@ object Boot extends IOApp with LazyLogging {
         conf.as[SwaggerConfig]("swagger")
       )
 
+      if (conf.getBooleanOption("backRawls").getOrElse(false)) {
+        logger.info("This instance has been marked as BACK. Booting monitors...")
+        BootMonitors.bootMonitors(
+          system,
+          conf,
+          slickDataSource,
+          gcsDAO,
+          samDAO,
+          pubSubDAO,
+          appDependencies.googleStorageService,
+          methodRepoDAO,
+          dosResolver,
+          workspaceServiceConstructor,
+          shardedExecutionServiceCluster,
+          maxActiveWorkflowsTotal,
+          maxActiveWorkflowsPerUser,
+          userServiceConstructor,
+          projectTemplate,
+          metricsPrefix,
+          requesterPaysRole,
+          useWorkflowCollectionField,
+          useWorkflowCollectionLabel,
+          defaultBackend,
+          methodConfigResolver,
+          GoogleProject(conf.getString("avroUpsertMonitor.pubSubProject")),
+          conf.getString("avroUpsertMonitor.pubSubTopic"),
+          conf.getString("avroUpsertMonitor.pubSubSubscription"),
+          conf.getString("avroUpsertMonitor.bucketName")
+        )
```

Comment:
so many arguments! Consider making a case class `AvroUpsertConfig` or whatever that encapsulates the pubsub project/topic/subscription and the bucket name, and pass that around?

Also, these will require settings in conf or fail, right?